internal/mysqld/parser/antlrparser/parser/MySqlLexer.g4

```diff
@@ -233,6 +233,11 @@ WRITE:                               'WRITE';
 XOR:                                 'XOR';
 ZEROFILL:                            'ZEROFILL';
 
+// ANNS
+ANNS:                                'ANNS';
+L2:                                  'L2';
```

Comment:
This placeholder does not need to distinguish metric_type, it is recommended to change the name to ANNS_FUNCTION（Candidate）